streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/kstream/KStream.scala

```diff
@@ -484,10 +472,10 @@ class KStream[K, V](val inner: KStreamJ[K, V]) {
    *                    one for each matched record-pair with the same key and within the joining window intervals
    * @see `org.apache.kafka.streams.kstream.KStream#leftJoin`
    */ 
-  def leftJoin[VO, VR](otherStream: KStream[K, VO],
-    joiner: (V, VO) => VR,
-    windows: JoinWindows)(implicit joined: Joined[K, V, VO]): KStream[K, VR] =
-      inner.leftJoin[VO, VR](otherStream.inner, joiner.asValueJoiner, windows, joined)
+  def leftJoin[VO, VR](otherStream: KStream[K, VO], windows: JoinWindows)(
+    joiner: (V, VO) => VR
```

Comment:
Ditto here.

Reply:
Indeed I forgot this one

Reply:
Hmm... this seems still not correct to me? Should it be

```
(otherStream: KStream[K, VO])(joiner: (V, VO) => VR, windows: JoinWindows)
```

Reply:
Now it's good